File: LV.1/두 개 뽑아서 더하기/solution.py
def solution(a):
    b=[]
    c=0
    a.sort()

    for i in range(len(a)):
        for j in range(i+1,len(a)):
            c=a[i]+a[j]
            b.append(c)
    b = set(b)
    b = list(b)
    b.sort()
    return b

File: LV.1/두 개 뽑아서 더하기/Solution.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        Arrays.sort(numbers);
        
        ArrayList<Integer> list = new ArrayList<> ();
        int total;

        for(int i=0; i<numbers.length-1; i++) {
            for(int j=i+1; j<numbers.length; j++) {
                total = numbers[i] + numbers[j];
                if(!list.contains(total)) {
                    list.add(total);
                }
            }
        }
        
        int[] answer = new int[list.size()];

        for(int i=0; i<list.size(); i++) {
            answer[i] = list.get(i);
        }
        return answer;
    }
}
